torch/quantization/convert_modules.py

```diff
@@ -69,7 +113,9 @@ def prepare(module, qconfig_dict):
         qconfig_dict: dictionary that maps from name of submodule to quantization
                       configuration
     """
-    return add_observer(module, qconfig_dict)
+    propogate_qconfig(module, qconfig_dict)
```

Comment:
I think we should flip the order here:
Zero: Propagate QconfigInfo
First call add_quant_dequant: This inserts quant and dequant stubs - no observers anywhere. QuantStub in this case is a pure placeholder does not contain observers.
Second: Insert observers: This would insert observers for both QuantStubs and for the module itself.
User is free to skip the add_quant_dequant step and do it manually. Again, QuantStub does not contain observers.

Reply:
sounds good

Reply:
yes, we should